src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -130,7 +130,7 @@
                                       Style="{StaticResource InCanvasScrollViewerStyle}"
                                       CanContentScroll="True"
                                       Width="250">
-                            <ContentControl Template="{StaticResource MemberControlTemplate}" />
+                            <ContentControl Template="{StaticResource InCanvasMemberControlTemplate}" />
```

Comment:
Yep that's exactly my point, @aosyatnik. Let's see if @lillismith has any view on this in terms of design... Lilli, here's what we're discussing here -- there are two scenarios in which a class member method is displayed: one in regular library view when user expands a given class, and another in search results (both library and in-canvas cases). In regular library view, a class member method includes its signature (e.g. `ByCoordinates(double x, double y)`) and in search view, a class member method does not include its signature (e.g. `ByCoordinates`).

Would you think we should align these two views to only show method names and not the full signatures? (If we decide to unify those two, we can do it as separate PR, not this one.)